### ISSUE TITLE
git can cause sniffer to go BONKERS

### ISSUE THREAD

**Issue by derek-adair** [jeffh/sniffer#28]
Perhaps there is nothing to be done about this, but there seems to be some funny behavior with certain git actions.  I rebased something the other day, and it re-ran my test suite about 4-5 times.

I'm running django 1.8, python 3.4 and selenium 2.45, i *believe* I am using the latest pypi sniffer (0.3.5).  I will continue to provide more meaningful details and perhaps compile a list of behaviors for you.  For now, this is all i got!

**Comment by jeffh** [comment]
Yeah. Sniffer currently doesn't ignore watching repository directory. Sounds like a great enhancement though.




—

**Comment by derek-adair** [comment]
The main symptom is when i rebase.  Something about when I am in the middle of squashing / merging.  This may not even be practical to switch.  Seems like I just may need to cancel the sniffer loop and restart it.

Seems like this may be a rather impractical enhancement :/

**Comment by jeffh** [comment]
Sniffer could support it by ignoring dotfiles (anything that prefixes with a period). I don't think that would be surprising behavior, but I could be wrong about that.

**Comment by derek-adair** [comment]
Jeez, i'd never thought of something that obvious and simple.  I'll probably figure out a way to manually ignore the .git dir for now.
